src/QuantumSdk/Sdk/Sdk.props

```diff
@@ -44,6 +46,22 @@
                       Include="Microsoft.Quantum.QirGeneration" Version="0.15.2101126940"
                       IsImplicitlyDefined="true" 
                       IsQscReference="true" ExecutionTarget="Any" Priority="-1" PrivateAssets="All" />
+    <PackageReference Condition="$(QirGeneration) And $([MSBuild]::IsOsPlatform('Windows'))"
+                      Include="libLLVM.runtime.win-x64" Version="11.0.0"
+                      IsImplicitlyDefined="true" 
+                      PrivateAssets="All" GeneratePathProperty="true" />
+    <PackageReference Condition="$(QirGeneration) And $([MSBuild]::IsOsPlatform('OSX'))"
+                      Include="libLLVM.runtime.osx-x64" Version="11.0.0"
+                      IsImplicitlyDefined="true" 
+                      PrivateAssets="All" GeneratePathProperty="true" />
+    <PackageReference Condition="$(QirGeneration) And $([MSBuild]::IsOsPlatform('Linux')) And '$(UbuntuVersion)' == '18.04'"
```

Comment:
I haven't found a good way to figure out what the ubuntu version is (or if we are on ubuntu), and it is also unfortunate that we have to explicitly reference the os version for linux. At least the UbuntuVersion property gives a way to make it work on 18.04 in addition to 20.04 which is the default.

Reply:
Yeah, this is an unfortunately necessity. From what I recall from building LLVM from source, there is a library provided by the OS that differs in some signatures from 18.04 to 20.04. While it probably isn't functionality we use directly, the LLVM binary will fail to load if it was compiled against one version but only the other version is present. In my experiments I found that I could install the newer library into 18.04 and get the LLVM compiled against 20.04 to work, but that seems like a weird dependency for an Ubuntu 18.04 user to have to manage. Further, I don't know what other workloads may get confused by having both versions of the library on the system, if any. More than that, I don't know if this library difference is something that would affect other versions of Linux such that we need to also grab the FreeBSD and SLES versions of the libLLVM package too (see https://github.com/microsoft/LLVMSharp/tree/a40e0ce6eefb2bed0b66a523611e306d5681e7cf/packages/libLLVM for the list of all native platforms the libLLVM package is produced for). I agree that for now supporting just Ubuntu 18.04 and Ubuntu 20.04 should probably be sufficient.